polls/admin.py

```diff
@@ -0,0 +1,7 @@
+from .models import Question
+from django.contrib import admin
+
+
+
+admin.site.register(Question)
```

Comment:
@fermoselle @Alejandroestmar Os aconsejo echarle un vistazo a la doc: https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#the-register-decorator

Ejemplo
```
from django.contrib import admin
from .models import Author

@admin.register(Author)
class AuthorAdmin(admin.ModelAdmin):
    pass
```